packit/utils.py

```diff
@@ -303,20 +304,34 @@ def rpmdev_bumpspec(
     run_command(cmd)
 
 
-def is_str_url(inp: str) -> bool:
-    """ is provided string a URL? """
+def git_remote_url_to_https_url(inp: str) -> str:
```

Comment:
Most of the parsing is covered here: https://github.com/packit-service/ogr/blob/master/ogr/parsing.py#L75

It would be nice to use that or add `https_url`/`ssh_url` properties to RepoUrl class. (In the future?)

Reply:
I'll add a TODO to merge that func with parse_git_repo from ogr, will that work for you?